sdk/translation/Azure.AI.Translation.Document/src/StatusSummary.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.Translation.Document.Models
+{
+    internal partial class StatusSummary
```

Comment:
this won't be needed as this type is internal and the suer will never know it exist :)

Reply:
Yea but I thought I might as well change it for consistency. Would you like me to revert this change?

Reply:
Yes. One less thing we will have to maintain in the future :)